jobs/v4/howto/basic_client_event.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package howto
+
+import (
+	"context"
+	"fmt"
+	"io"
+  "time"
+
+	talent "cloud.google.com/go/talent/apiv4beta1"
+  "github.com/golang/protobuf/ptypes"
+	talentpb "google.golang.org/genproto/googleapis/cloud/talent/v4beta1"
+)
+
+// [START job_search_create_client_event]
+
+// createClientEvent creates a client event.
+func createClientEvent(w io.Writer, projectId string, requestId string, eventId string, relatedJobNames []string) (*talentpb.ClientEvent, error) {
+	ctx := context.Background()
+
+	// Create an eventService client.
+	c, err := talent.NewEventClient(ctx)
+	if err != nil {
+		fmt.Printf("talent.NewEventClient: %v", err)
+		return nil, err
+	}
+
+  createTime, _ := ptypes.TimestampProto(time.Now())
+  clientEventToCreate := &talentpb.ClientEvent{
```

Comment:
I think we can inline this declaration when creating `req` below. Can leave here if you prefer.